definitions/contested/json/CaseField/CaseField.json

```diff
@@ -3033,5 +3033,249 @@
     "FieldType": "Text",
     "Label": "Please Specify",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplicationDirectionsHearingRequired",
+    "FieldType": "YesOrNo",
+    "Label": "Is a Hearing required?",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplicationDirectionsHearingDate",
+    "FieldType": "Date",
+    "Label": "Hearing Date",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplicationDirectionsHearingTime",
+    "FieldType": "Text",
+    "Label": "Hearing Time",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplicationDirectionsHearingTimeEstimate",
+    "FieldType": "Text",
+    "Label": "Time Estimate",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/09/2020",
+    "CaseTypeID": "FinancialRemedyContested",
+    "ID": "generalApplicationDirections_regionList",
```

Comment:
could we re-use the court fields from the create journey? i.e. would BA/PO be happy that these values are pre-populated and would override previous select. want to avoid having to create all of these new fields to maintain